.github/workflows/kubeapps.yaml

```diff
@@ -0,0 +1,778 @@
+          cp ${HOME}/.kube/kind-config-kubeapps-ci ${HOME}/.kube/config
+          kubectl config set-context kind-kubeapps-ci
+
+          # If the default IP the proper one, the multicluster setup will fail
+          if [ "$DEFAULT_DEX_IP" != "$DEX_IP" ]; then echo "Default IP does not match with current IP used in Kind"; exit 1; fi
+
+          echo "DEFAULT_DEX_IP=${DEFAULT_DEX_IP}" >> $GITHUB_ENV
+          echo "DEX_IP=${DEX_IP}" >> $GITHUB_ENV
+          echo "ADDITIONAL_CLUSTER_IP=${ADDITIONAL_CLUSTER_IP}" >> $GITHUB_ENV
+      - name: "Install mkcert"
+        run: |
+          curl -LO "https://github.com/FiloSottile/mkcert/releases/download/${MKCERT_VERSION}/mkcert-${MKCERT_VERSION}-linux-amd64"
+          chmod +x "mkcert-${MKCERT_VERSION}-linux-amd64"
+          sudo mv "mkcert-${MKCERT_VERSION}-linux-amd64" /usr/local/bin/mkcert
+          mkcert -install
+      - name: "Install helm (minimum and stable)"
+        run: |
+          wget https://get.helm.sh/helm-${HELM_VERSION_MIN}-linux-amd64.tar.gz
+          tar zxf helm-$HELM_VERSION_MIN-linux-amd64.tar.gz
+          sudo mv linux-amd64/helm /usr/local/bin/
+
+          wget https://get.helm.sh/helm-${HELM_VERSION_STABLE}-linux-amd64.tar.gz
+          tar zxf helm-$HELM_VERSION_STABLE-linux-amd64.tar.gz
+          sudo mv linux-amd64/helm /usr/local/bin/helm-stable
+      - name: "Load needed images into Kind"
+        run: |
+          ./script/load-kind-image.sh docker.io/bitnami/apache:2.4.48-debian-10-r74 kubeapps-ci kubeapps-ci-additional &&
+          ./script/load-kind-image.sh docker.io/bitnami/apache:2.4.48-debian-10-r75 kubeapps-ci kubeapps-ci-additional &&
+          ./script/load-kind-image.sh registry:$DOCKER_REGISTRY_VERSION kubeapps-ci kubeapps-ci-additional
+      - name: "Download docker images"
+        uses: actions/download-artifact@v3
+        with:
+          path: /tmp/images
+      - name: "Load CI images in the cluster"
+        run: |
+          source ./script/lib/liblog.sh
+          for path in /tmp/images/*; do 
+            image=$(basename "$path")
+            info "Loading image ${image}"
+            kind load image-archive "${path}/${image}.tar" --name kubeapps-ci; 
+          done
+      - name: "Install multicluster deps"
+        run: |
+          sed -i -e "s/172.18.0.2/$DEFAULT_DEX_IP/g;s/localhost/kubeapps-ci.kubeapps/g" ./site/content/docs/latest/reference/manifests/kubeapps-local-dev-dex-values.yaml
+          helm repo add dex https://charts.dexidp.io
+
+          # Install dex
+          kubectl create namespace dex
+          helm install dex dex/dex --version 0.5.0 --namespace dex --values ./site/content/docs/latest/reference/manifests/kubeapps-local-dev-dex-values.yaml
+
+          # Install openldap
+          helm repo add stable https://charts.helm.sh/stable
+          kubectl create namespace ldap
+          helm install ldap stable/openldap --namespace ldap
```

Comment:
Same as above, with `make deploy-openldap`.

Reply:
What about just creating a script `install-multicluster-deps.sh`? After taking a look, I can see that this block of code is just used once in the whole CI (in the CircleCI one, I mean), so I don't think it would be really useful to have `deploy-dex` and `deploy-openldap` make targets, and I would prefer not to bloat the makefile. WDYT?

Reply:
Both `deploy-dex` and `deploy-opendldap` already exist in the [dev makefile](https://github.com/vmware-tanzu/kubeapps/blob/main/script/makefiles/deploy-dev.mk#L10-L22). My comment was towards reusing them in the CI, if possible. Otherwise, fine with leaving them in the GH Yaml.

Reply:
Ok, I see... For `deploy-dex` I think we could reuse it as is. However, for `deploy-openldap`, the make target is calling Helm providing the values file in `./site/content/docs/latest/reference/manifests/kubeapps-local-dev-openldap-values.yaml`, while the scripting version in the CI is installing it without providing custom values, so I don't know what would imply that change...